### PR TITLE
ref(debug-files): Add custom repositories new design

### PR DESCRIPTION
**Pull request by priscilawebdev**
. Updates debug files design

https://www.figma.com/file/cF3qSQA5urBHvKR8ww1EK9/App-Store-Connect?node-id=33%3A3149

**Preview:**

blah2

INGEST-178

### CODE REVIEW

File: static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          tct('v[version]', {version: latestBuildVersion})
+        ) : (
+          <NotAvailable tooltip={t('Not available')} />
+        )}
+      </Value>
+
+      {t('Last detected build')}
+      <Value>{latestBuildNumber ?? <NotAvailable tooltip={t('Not available')} />}</Value>
+
+      {/* {t('Detected last build on')}
```
Comment (priscilawebdev) [inline static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx:29]: The comments will be removed as soon as the backend provides the correct data format

File: static/app/views/settings/projectDebugFiles/externalSources/customRepositories/status.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        <IconWrapper>
+          <IconDownload size="sm" />
+        </IconWrapper>
+        {tn('%s build pending', '%s builds pending', pendingDownloads)}
+      </Wrapper>
+    );
+  }
+
+  if (lastCheckedBuilds) {
+    // return (
```
Comment (priscilawebdev) [inline static/app/views/settings/projectDebugFiles/externalSources/customRepositories/status.tsx:88]: same here

File: static/app/views/settings/projectDebugFiles/externalSources/buildInRepositories.tsx [hunk 3]
```diff
@@ -14,14 +15,20 @@ type Props = {
   builtinSymbolSources: string[];
 };
 
-function BuildInSymbolSources({
+function BuildInRepositories({
```
Comment (matejminar) [inline static/app/views/settings/projectDebugFiles/externalSources/buildInRepositories.tsx:18]: Should this be `BuiltInRepositories`?

File: static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
+
+// import DateTime from 'app/components/dateTime';
```
Comment (matejminar) [inline static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx:3]:
```suggestion
```

**Review by matejminar — APPROVED** [review]

File: static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx (hunk 4 above)
Comment (priscilawebdev) [reply static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx:3]: I will enable this as soon as the backend provides the correct data format

**Pull request merged**